packaging/msi/Product.wxs

```diff
@@ -0,0 +1,111 @@
+        InstallerVersion="500"
+        Compressed="yes"
+        UpgradeCode="DDC28522-EF11-4D6C-9F53-B76DF967C8B2">
+
+        <!-- Embed CAB files in MSI -->
+        <MediaTemplate EmbedCab="yes" />
+
+        <!-- Define the product icon -->
+        <Icon Id="icon.ico" SourceFile="!(bindpath.SOURCE_ROOT)\packaging\msi\resources\product.ico"/>
+        <Property Id="ARPPRODUCTICON" Value="icon.ico" />
+
+        <!-- Use the default WixUI_InstallDir dialog set and personalize the license an images -->
+        <ui:WixUI Id="WixUI_InstallDir" InstallDirectory="INSTALLFOLDER" />
+        <WixVariable Id="WixUILicenseRtf" Value="!(bindpath.SOURCE_ROOT)\packaging\msi\docs\LICENSE.rtf"/>
+        <WixVariable Id="WixUIBannerBmp" Value="!(bindpath.SOURCE_ROOT)\packaging\msi\resources\banner.jpg"/>
+        <WixVariable Id="WixUIDialogBmp" Value="!(bindpath.SOURCE_ROOT)\packaging\msi\resources\dialog.jpg"/>
+
+        <!-- Define the directory structure -->
+        <StandardDirectory Id="ProgramFiles64Folder">
+            <Directory Id="INSTALLFOLDER" Name="ZeroC\Ice-3.8.0">
+                <Directory Id="BinFolder" Name="bin"/>
+                <Directory Id="SliceFolder" Name="slice"/>
+                <Directory Id="ConfigFolder" Name="config"/></Directory>
+        </StandardDirectory>
+        <StandardDirectory Id="ProgramMenuFolder">
+            <Directory Id="MyAppStartMenu" Name="Ice 3.8.0"/>
+        </StandardDirectory>
+
+        <!-- Define the components that install the require files -->
+        <ComponentGroup Id="ProductComponents">
+            <!-- Binary files from the x64 Release build -->
+            <Files Include="!(bindpath.SOURCE_ROOT)\cpp\bin\x64\Release\**" Directory="BinFolder"/>
+
+            <!-- The IceGrid GUI jar -->
+            <File Source="!(bindpath.SOURCE_ROOT)\java\lib\icegridgui.jar" Directory="BinFolder" />
+
+            <!-- Slice Files-->
+            <Files Include="!(bindpath.SOURCE_ROOT)\slice\**" Directory="SliceFolder"/>
+
+            <!-- Config Files -->
+            <Files Include="!(bindpath.SOURCE_ROOT)\cpp\config\*.cfg" Directory="ConfigFolder"/>
+            <Files Include="!(bindpath.SOURCE_ROOT)\cpp\config\templates.xml" Directory="ConfigFolder"/>
+
+            <!-- License Files -->
+            <File Id="License" Source="!(bindpath.SOURCE_ROOT)\packaging\msi\docs\LICENSE.txt" />
+            <File Id="IceLicense" Source="!(bindpath.SOURCE_ROOT)\packaging\msi\docs\ICE_LICENSE.txt" />
+            <File Id="ThirdPartyLicense" Source="!(bindpath.SOURCE_ROOT)\packaging\msi\docs\THIRD_PARTY_LICENSE.txt" />
+        </ComponentGroup>
+
+        <!-- Define the components that install the start menu shortcuts -->
+        <ComponentGroup Id="StartMenuComponents">
+            <Component
+                Id="AppStartMenuShortcuts"
+                Directory="MyAppStartMenu"
+                Guid="618E2B68-1829-478A-AB46-47D65887F351">
+                <util:InternetShortcut
```

Comment:
Should we keep all of these start menu links? Seems a bit over the top to me.

Reply:
I'm fine removing some, we can decide later which ones too keep.